### PR TITLE
Add Receiver fixes

### PR DESCRIPTION
**Pull request by pierDipi**
Bring receiver fixes from 
https://github.com/rvansa/eventing-hyperfoil-benchmark
to this repo.

### CODE REVIEW

**Comment by rvansa** [comment]
@pierDipi Great, thanks!

Btw. did you have any issues with the header `ce-runId` vs. `ce-runid`? `Request.getHeader(...)` should not be case-sensitive (and proxies are free to rewrite case in headers).

**Comment by pierDipi** [comment]
> Btw. did you have any issues with the header `ce-runId` vs. `ce-runid`? `Request.getHeader(...)` should not be case-sensitive (and proxies are free to rewrite case in headers).

I didn't check if that could cause issues (likely it doesn't) but I'd leave it as in your repository since it's what we're currently using.

File: vertx-receiver/src/main/java/io/openshift/serverless/knative/eventing/hyperfoil/benchmark/VertxReceiverVerticle.java [hunk 1]
```diff
@@ -1,61 +1,64 @@
...
-   */
-  private final static String METRIC_SUFFIX_ENV = "METRIC_SUFFIX";
+  private final static long GARBAGE_COLLECTION_PERIOD = Long.getLong("receiver.gc.period", 60000);
+  private final static long COMPLETION_DELAY_REQUEST_PERIOD = Long.getLong("receiver.gc.period", 5000);
 
   private final static String CE_BENCHMARK_TIMESTAMP_EXTENSION = "ce-benchmarktimestamp";
+  private final static String CE_PHASE_START_TIMESTAMP = "ce-phasestart";
   private final static String CE_PHASE = "ce-phase";
-  private final static String CE_RUNID = "ce-runid";
+  private final static String CE_RUNID = "ce-runId";
```
Comment (matzew) [inline vertx-receiver/src/main/java/io/openshift/serverless/knative/eventing/hyperfoil/benchmark/VertxReceiverVerticle.java:29]: is that a change between versions?

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**